docs/TASKS.md

```diff
@@ -1,11 +1,31 @@
 Tasks
 =====
 
-A task describes the how, what, and when to do for a Snap job.  A task is described in a task _manifest_, which can be either JSON or YAML<sup>1</sup>.
+A task describes the how, what, and when to do for a Snap job.  A task is described in a task _manifest_, which can be either JSON or YAML<sup>1</sup>. The manifest can be divided into two parts: Header and Workflow.
+
+A task can be in the following states:
+- **running:** a running task
+- **stopped:** a task that is not running
+- **disabled:** a task in a state not allowed to start. This happens when the task produces consecutive errors. A disabled task must be re-enabled before it can be started again. 
+
+
+![newtaskstatediagram](https://cloud.githubusercontent.com/assets/21182867/19250638/4d78384a-8ef0-11e6-91db-2bd9288cf9b5.png)
```

Comment:
Since tasks can be created into the stopped state, should we represent that on this diagram?